### PR TITLE
HCF-1011: Build packages across different releases in parallel

### PR DESCRIPTION
**Pull request by Mook-as**

### CODE REVIEW

File: compilator/compilator.go [hunk 1]
```diff
@@ -47,7 +47,8 @@ type Compilator struct {
 	BaseType         string
 	FissileVersion   string
 
-	packageDone   map[string]chan struct{}
+	// packageDone is a map of release name -> package name -> channel to close when done
+	packageDone   map[string]map[string]chan struct{}
```
Comment (viovanov) [inline compilator/compilator.go]: do we need a 2 dimentional map?
can we identify packages by their signature?

File: compilator/compilator.go [hunk 2]
```diff
@@ -682,9 +698,12 @@ func (c *Compilator) BaseImageName() string {
 	return util.SanitizeDockerName(fmt.Sprintf("%s:%s", c.baseCompilationImageRepository(), c.baseCompilationImageTag()))
 }
 
-func (c *Compilator) initPackageMaps(release *model.Release) {
-	for _, pkg := range release.Packages {
-		c.packageDone[pkg.Name] = make(chan struct{})
+func (c *Compilator) initPackageMaps(releases []*model.Release) {
+	for _, release := range releases {
```
Comment (viovanov) [inline compilator/compilator.go]: Here we can range over all packages in all releases and just dump all of them in map, referencing them by their signature.

File: compilator/compilator.go (hunk 1 above)
Comment (Mook-as) [inline compilator/compilator.go]: Addressed in 0c7329d5556d5358baeff2f3bd5d39538bf8ab8b

**Comment by mallow111** [comment]
Can someone help to delete my account from this repo please?
Thanks!

**Comment by aarondl** [comment]
@mallow111 I looked, you're not directly attached to this repo in any way other than through the org just like everyone else?

**Comment by mallow111** [comment]
Hi Guys,

I am not working for team anymore, not sure how to remove my account by
myself, sorry for the inconvenience and thanks a lot!

Min

**Comment by aarondl** [comment]
@mallow111 I double checked. You're not on any org/team that would make you get e-mails for this repo. If you look at the original e-mail at the bottom, Github describes why you're seeing the notification. You can simply mute this thread to stop getting e-mails. Check out https://github.com/notifications as well to configure notifications.

**Comment by mallow111** [comment]
Will do, thank you guys!

**Pull request merged**